ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -18,6 +18,17 @@
 ####      See model_hvac.md for detailed definitions      ####
 ##############################################################
 
+
+TRU:
+  id: "oli0001"
+  description: "Tag for trench heating units. Trench Heating units are in floor airconditioning units. 
```

Comment:
Clarify "particular to under-floor air distribution systems" or something similar.